app2.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+var storeHourArray = ['','6am','7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm','Total'];
+var storeName = ['1st and Pike','SeaTac','Seattle Center','Capitol Hill','Alki'];
+var body = document.getElementsByTagName('body')[0];
+
+function cookiesStore(name,maxcustomer,mincustomer,avgCookieHour) {
```

Comment:
JavaScript naming convention is to use camelCase (e.g. maxCustomer, minCustomer)